### PR TITLE
SDIT-776: ✨ Add in business hours support for queues

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: charts/generic-prometheus-alerts/values.yaml [hunk 1]
```diff
@@ -97,3 +97,6 @@ rdsAlertsConnectionThresholdMinutes: 5
 
 # Set the error window minutes for 5xx ingress
 ingress5xxErrorWindowMinutes: 1
+
+# Alert for business hours only - 7am to 6pm UTC (8am to 7pm BST) on week days (Monday to Friday)
+businessHoursOnly: false
```
Comment (petergphillips) [inline charts/generic-prometheus-alerts/values.yaml:102]: default is for all hours

File: charts/generic-prometheus-alerts/templates/aws-sqs-alerts.yaml [hunk 2]
```diff
@@ -18,14 +19,21 @@ spec:
   groups:
     - name: {{ $targetApplication }}-sqs
       rules:
+        - record: business_day
+          expr: 0 < day_of_week() < 6
+        - record: business_hour
+          expr: 7= < hour() < 18
```
Comment (petergphillips) [inline charts/generic-prometheus-alerts/templates/aws-sqs-alerts.yaml:25]: between 7am and 6pm (8am to 7pm BST)

Comment (steverendell) [reply charts/generic-prometheus-alerts/templates/aws-sqs-alerts.yaml:25]: "7=" looks odd, does this mean inclusive?

Comment (petergphillips) [reply charts/generic-prometheus-alerts/templates/aws-sqs-alerts.yaml:25]: 🤦

**Pull request merged**